google/cloud/bigtable_v2/types/bigtable.py

```diff
@@ -139,7 +139,7 @@ class RequestStatsView(proto.Enum):
         number=6,
         enum=RequestStatsView,
     )
-    reversed_: bool = proto.Field(
+    reversed: bool = proto.Field(
```

Comment:
This is not yet released (See https://github.com/googleapis/python-bigtable/pull/827)